src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -177,6 +186,10 @@ private void validateSchema(final EventType eventType) throws InvalidEventTypeEx
         }
     }
 
+    private void assignTopic(final EventType eventType) {
+        eventType.setTopic(uuidGenerator.randomUUID().toString());
```

Comment:
In order to do so, correctly, we should add some more limitations to the event type name creation and handle those already created that doesn't fit in this constraints.